main.py

```diff
@@ -90,13 +95,15 @@ def install_patches(categories, with_interactive):
     try:
         logging.info("Installing patches...")
         output = subprocess.check_output(command)
+        update_occurred = True
```

Comment:
Would need to check if there actually were any updates to install. Currently it just runs refresh and install and does not care if there actually is anything to install, so this would just set update_occured always to True (unless there was an error)